caprover/INSTALL_GC_STACK.md

```diff
@@ -24,8 +24,7 @@ From the CapRover, navigate to **Apps** and "One-Click Apps/Database". Find the
 
 If you plan to expose the database to applications not hosted on this VM's CapRover,
 you will need to take some additional steps after installing the one-click-app:
-- Uncheck "Do not expose as web-app externally".
-- Enable and force HTTPS.
+- Make sure "Do not expose as web-app externally" is checked.
```

Comment:
I don't think this follows the "if" clause just preceding it. These are additional steps in to support the optional case when you want to expose the DB to the outside internet.

Reply:
Yes, this **is** the correct behavior to support that use case. 

It does not seem obvious, but if "Do not expose as web-apps externally" == false, then trying to connect to postgres.{domain} does not work. I think it is because CapRover then routes it via Nginx, so only port 80 / 443.

Whereas if "Do not expose as web-apps externally" == true, then you can access Postgres on the domain via route 5432, providing it is mapped server-to-container. I know this because I tried it and it works.

Unless you meant something else?